config/environments/production.rb

```diff
@@ -118,6 +118,8 @@
     ActionMailer::Base.add_delivery_method :ses, AWS::SES::Base,
                                            access_key_id: ENV['AWS_ACCESS_KEY_ID'],
                                            secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
+                                           server: "email.#{ENV['SES_REGION']}.amazonaws.com",
+                                           message_id_domain: "#{ENV['SES_REGION']}.amazonses.com",
```

Comment:
we should have this as an optional parameter, maybe use fetch ENV.fetch() to give the defaults to us-east-1 , this way we will not break existing installations